src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -600,6 +600,17 @@ def launch_fluent(
 
     del start_container
 
+    # Check if user set PYFLUENT_ADDITIONAL_ARGUMENTS var
+    if os.environ.get("PYFLUENT_ADDITIONAL_ARGUMENTS", None) is not None:
```

Comment:
```suggestion
    if os.getenv("PYFLUENT_ADDITIONAL_ARGUMENTS"):
```

Reply:
Please also update the function docstring

Reply:
I has been updated